site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig

```diff
@@ -4,6 +4,8 @@
 {% for script in localjs %}
     <script src="{{ script }}"></script>
 {% endfor %}
+<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>  
```

Comment:
why is jQuery 1.12.4 here? Does jspdf work on newer versions of jQuery?

Reply:
Yes it does work with newer versions. Removed the line